internal/clients/idler.go

```diff
@@ -51,45 +78,52 @@ func NewIdler(url string) IdlerService {
 	}
 }
 
-// IsIdle returns true if the Jenkins instance for the specified tenant is idled, false otherwise.
-func (i *idler) IsIdle(tenant string, openShiftAPIURL string) (bool, error) {
+// State returns the state of Jenkins instance for the specified tenant
+func (i *idler) State(tenant string, openShiftAPIURL string) (podState, error) {
```

Comment:
What's the incentive behind using podState instead of string?

Reply:
It is a string but type aliased as podState so that golang can type check it e.g.
the code below produces error. 

```go
package main

type Color string

const (
	Red Color = "red"
)

type Name string

const (
	Foobar Name = "foobar"
)

func main() {
	var x Color = Foobar // ERROR
}

```